tests/cupy_tests/core_tests/fusion_tests/fusion_utils.py

```diff
@@ -0,0 +1,124 @@
+import numpy
+import six
+
+import cupy
+from cupy import testing
+
+
+scalar_types = (numpy.generic, float, complex) + six.integer_types
+
+
+def check_fusion(
+        generate_inputs_name='generate_inputs',
+        generate_inputs_args=None,
+        check_array=None,
+        check_array_kwargs=None,
+        accept_error=()):
+    """Decorator for tests for ``cupy.fuse``.
+
+    This decorator checks
```

Comment:
?

Reply:
Sorry, I have forgotten to complete this comment.